cirq-google/cirq_google/transformers/target_gatesets/google_cz_gateset.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict, List, Sequence, Type, Union
+import cirq
+
+
+class GoogleCZTargetGateset(cirq.CZTargetGateset):
+    """CZTargetGateset implementation tailored to Google devices."""
```

Comment:
I would prefer a more explanatory test.  This is a fine one-line summary, but I would like to see a further description, such as why it is tailored to Google devices.